scripts/quantitative_testing.py

```diff
@@ -26,8 +27,9 @@
 def closest_word(word: str, words: List[str]) -> str:
     edit_distances = [nltk.edit_distance(word, w) for w in words]
     for i, _ in enumerate(edit_distances):
-        if words[i].lower()[:2] == word.lower()[:2]:
-            edit_distances[i] -= 5
+        # Discount if we are using a unique common prefix.
+        common_prefix = os.path.commonprefix([word.lower(), words[i].lower()])
```

Comment:
Whoa, using prefixes instead of hardcoded matches? Very cool, love to see it.

Reply:
overengineering at its finest